README.md

```diff
@@ -16,7 +16,7 @@ Add the following dependency to your project
 ```xml
 <dependency>
   <groupId>io.camunda</groupId>
-  <artifactId>zeebe-bpmn-assert</artifactId>
+  <artifactId>zeebe-process-test</artifactId>
```

Comment:
:x: Please also consider renaming top level heading in Readme. And maybe also rephrasing the first paragraph Camunda Cloud vs. Zeebe. 

:x: There is also this line in the readme.md which needs to be updated
> For example tests the best place to look right now is the tests in io.camunda.zeebe.bpmnassert.assertions

And maybe do a general text search for the old name to see if anything else remained

Reply:
:clown_face: Love the avatar, by the way :clown_face:

Reply:
Thanks 😄 

I've fixed the README. The general text search yielded no results apart from the Vault which I mentioned before. That'll probably be fixed in a separate PR tomorrow.